libs/dh/shared/data-access-graphql/src/lib/error-handler.ts

```diff
@@ -19,7 +19,6 @@
 import { onError } from '@apollo/client/link/error';
 
 import { DhApplicationInsights } from '@energinet-datahub/dh/shared/util-application-insights';
-import { translate } from '@jsverse/transloco';
```

Comment:
Strange, that it didnt get caught by the lint

Reply:
Apparently it did ... but a little too late.

On "main" 👇

![image](https://github.com/user-attachments/assets/a510d188-a2cf-4b6e-9a1b-c9bdba3971f9)